bigframes/display/anywidget.py

```diff
@@ -53,8 +53,8 @@
 
 @dataclasses.dataclass(frozen=True)
 class _SortState:
-    column: str
-    ascending: bool
+    columns: tuple[str, ...]
+    ascending: tuple[bool, ...]
```

Comment:
I suspect that this data structure allows content that make little sense in practice. For example:

```
columns: ["my_col", "my_col"]
ascending: [True, False]
```

Shall we define a `__post_init__()` method that performs validation? I.e. if a column name appears multiple time in the tuple, raise an error?